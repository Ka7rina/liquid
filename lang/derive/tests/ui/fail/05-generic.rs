use liquid_lang as liquid;

use liquid::InOut;
#[derive(InOut)]
pub struct MyStruct<T> {
    a: T,
}

fn main() {}
