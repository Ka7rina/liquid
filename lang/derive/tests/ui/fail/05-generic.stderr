error: generic structs are not supported to derive from `InOut` or `State`
 --> $DIR/05-generic.rs:5:20
  |
5 | pub struct MyStruct<T> {
  |                    ^^^
